draft-ietf-httpbis-digest-headers.md

```diff
@@ -466,19 +466,27 @@ in accordance with the policies set out in this section.
 Each algorithm has a status field, which is intended to provide an aid to
 implementation selection.
 
-Algorithms with a status value of "standard" are suitable for many purposes,
-including adversarial situations where hash functions might need to provide
-resistance to collision, first-preimage and second-preimage attacks. For
-adversarial situations, selecting which of the "standard" algorithms are
-acceptable will depend on the level of protection the circumstances demand. As
-there is no negotiation, endpoints that depend on a digest for security will be
-vulnerable to attacks on the weakest algorithm they are willing to accept.
+Algorithms with a status value of "standard" are suitable for many purposes and
+it is RECOMMENDED that applications use these algorithms. These can be used in
+adversarial situations where hash functions might need to provide resistance to
+collision, first-preimage and second-preimage attacks. For adversarial
+situations, selecting which of the "standard" algorithms are acceptable will
+depend on the level of protection the circumstances demand. As there is no
+negotiation, endpoints that depend on a digest for security will be vulnerable
+to attacks on the weakest algorithm they are willing to accept.
```

Comment:
WRT this last sentence repeats what we already state in:

1. [This I-D does not define means for authentication, authorization, or privacy](https://www.ietf.org/archive/id/draft-ietf-httpbis-digest-headers-12.html#section-1.2-9)
2. [Security considerations](https://www.ietf.org/archive/id/draft-ietf-httpbis-digest-headers-12.html#section-6.6-3). 

Probably we can move them there.

Reply:
yeah I'll take that PR instead of this hunk